client/src/main/java/io/pravega/client/stream/notifications/events/EndOfDataEvent.java

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream.notifications.events;
+
+/**
+ * Class to represent a end of data event. This event is generated when all the streams(sealed) managed by the reader
+ * group are completely read by the readers.
+ */
+public class EndOfDataEvent extends Event {
```

Comment:
While reviewing this, I noticed that the use of the term "event" is overloaded in our API. Is it worth changing the term "event" in this context to "notification" to disambiguate?

If we do it, then we need to change for segment event as well, so we could do it in a separate PR.

Reply:
Refactored the code to use the term notification.